modules/crypto/src/lib/utils/base64-utils.d.ts

```diff
@@ -0,0 +1,12 @@
+export function toHex(cipher: number): string;
```

Comment:
How about rename this file to `encoding-utils` since we can potentially add more encoding functions and `toHex` isn't really a base64 related function.